Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py

```diff
@@ -0,0 +1,795 @@
+    '''
+    incidents = []
+    for vuln in vulns:
+        # get vulnerability active view (av) date created values:
+        av_date_created_str = vuln.get('active_view_date_created')
+        av_date_created_dt = datetime.strptime(av_date_created_str, "%Y-%m-%dT%H:%M:%S.%fZ")
+
+        # Skip if vuln created before current start_time:
+        if av_date_created_dt < start_time_dt:
+            continue
+        vuln_event = create_vuln_event_object(vuln)
+        incident = {
+            'name': vuln.get('title'),
+            'occurred': vuln.get('active_view_date_created'),
+            'details': vuln.get('data'),
+            'rawJSON': json.dumps(vuln_event)
+        }
+        incidents.append(incident)
+    return incidents
+
+
+def fetch_incidents():
+    ''' Method to fetch Demisto incidents by pulling any new vulnerabilities found.   '''
+    try:
+        last_run = demisto.getLastRun()
+        # Check if last_run exists and has a start_time, else create new start_time timestamp:
+        if last_run and last_run.get('start_time', False):
+            start_time = last_run.get('start_time')
+            # Check if user selected fetch frequency is within last_run[start_time]:
+            fetch_frequency_td = get_fetch_frequency_td()
+            if (datetime.utcnow() - datetime.utcfromtimestamp(int(start_time))) < fetch_frequency_td:
+                debug_msg = "frequency (" + str(fetch_frequency_td) + ") is within last start_time, sending empty incidents."
+                demisto.debug("FrontlineVM fetch_incidents -- " + debug_msg)
+                demisto.setLastRun({'start_time': last_run.get('start_time')})
+                demisto.incidents([])
+                return
+            debug_msg = "frequency (" + str(fetch_frequency_td) + ") exceeds last start_time, getting incident events."
+            demisto.debug("FrontlineVM fetch_incident -- " + debug_msg)
+        else:
+            now = datetime.utcnow()
+            start_time = now.strftime("%s")
+        start_time_dt = datetime.utcfromtimestamp(int(start_time))
+        start_time_str = start_time_dt.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+        demisto.debug("FrontlineVM fetch_incident -- continuing with start_time of " + str(start_time_str))
+
+        # Fetch vulnerabilities:
+        req_params = {}
+        req_params['lte_vuln_severity_ddi'] = str(INCIDENT_VULN_MIN_SEVERITY)
+        req_params['gte_vuln_date_created'] = start_time_str
+        req_params = parse_params(req_params)
+        vulns = get_fvm_data(VULN_ENDPOINT, params=req_params)
+
+        # Create incidents
+        if vulns:
+            demisto.debug("FrontlineVM fetch_incidents -- getting all incidents.")
+            incidents = vulns_to_incident(vulns, start_time_dt)
+        else:
+            demisto.debug("FrontlineVM fetch_incidents -- no new vulnerabilities found, no incidents created.")
+            incidents = []
+
+        now = datetime.utcnow()
```

Comment:
maybe if incidents were found, you should make the time of your most recent vulnerability the start_time, since maybe some vulnerabilities found between that time and the current time haven't been indexed yet

Reply:
You're correct.
Updated the API GET request when pulling vulnerabilities to order the vulnerabilities by the time they were created. 
Will be using the last vulnerability in the list to reference the new start_time.